jekyll/_components/tables.md

```diff
@@ -2,17 +2,142 @@
 layout: page
 title: Tables
 order: 9
-draft: true
 ---
 
-Tabular data appears in a variety of places, the Network panel is the most obvoious example. The console is also able to output a visual table using `console.table()` e.g. `console.table(["apples", "oranges", "bananas"]);`
-
-![console table example](../images/components/tables/console-table-example.png "Console table example")
+Tabular data appears in a variety of places, the Network panel is the most obvious example. The console is also able to output a visual table using `console.table()` e.g. `console.table(["apples", "oranges", "bananas"]);`
 
 ## Usage
 
+<div class="grid-2" markdown="1">
+![Usage Example of a Checkbox as used in Firefox Options](../images/components/tables/table-grid-light.png)
+
+<div markdown="1">
+Tables are used when data needs to be presented in a series of rows which may also posses any amount of columns. For large quantities of data it is recommended to include a button column header allowing users to sort the data by that specific column, as seen in the Network panel
+</div>
+</div>
+
+
 ## Types
+
+### Console tables
+
+<div class="grid-2" markdown="1">
+![Console table example](../images/components/tables/console-table-small.png)
+
+<div markdown="1">
+The console can display a tablular representation of an Array or Object. e.g. `console.table(["apples", "oranges", "bananas"]);`
+
+Console table columns are not sortable or resizable.
+</div>
+</div>
+
+### Network and Application panel
     
+<div class="grid-2" markdown="1">
+![Network table timeline example](../images/components/tables/table-timeline.png)
+
+<div markdown="1">
+The Network panel table features column sorting and row highlighting on hover. the table can contain a rich variety of data including text, images (icons: `16px` x `16px`) and timeline interface elements like graphs and vertical rules (pictured).
+
+Filter controls are provided in the form of buttons for specific resource types and a filter input field for filtering URLs.
+</div>
+</div>
+
 ## Styles
 
+Table rows alternate their background color, the subtle "zebra" effect assists in legibility:
+
+### Dark mode
+<div class="grid-2" markdown="1">
+![Usage Example of a Checkbox as used in Firefox Options](../images/components/tables/table-grid-dark.png)
+
+<div markdown="1">
+Odd rows: Grey 80 `#2a2a2e`
+
+Even rows: `rgba(255,255,255,0.05)`
+
+Column header: Grey 85 `#1b1b1d`
+
+Borders/splitters: `#3c3c3d`
+
+Text: Grey 40 `#b1b1b3`
+
+Selected row: `#204E8A`
```

Comment:
We should probably make a separate section for the selected row, [like this](http://localhost:4000/photon/components/checkboxes.html#on--off), and add the text colour there, too.